### PR TITLE
Add RLCraft example

### PR DESCRIPTION
**Pull request by brutalgg**
Adding an RLCraft compose example for future reference.

### CODE REVIEW

**Comment by brutalgg** [comment]
I saw that #996 had been created but it wasn't finished and I happened to need this exact compose file so I created it and figured I'd share if it was valuable.

File: examples/docker-compose-rlcraft.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+services:
+  rlcraft:
+    image: itzg/minecraft-server:java8
+    container_name: rlcraft
+    volumes:
+    - rlcraft-modpack:/modpacks:ro
+    - rlcraft-data:/data
+    environment:
+      EULA: "true"
+      TYPE: "FORGE"
```
Comment (itzg) [inline examples/docker-compose-rlcraft.yml:12]: TYPE needs to be "CURSEFORGE" to make use of CF_SERVER_MOD variable.

Comment (brutalgg) [reply examples/docker-compose-rlcraft.yml]: Fixed in a1e4203

**Comment by brutalgg** [comment]
Seems making that changed has now caused some problems. Going to try and troubleshoot a fix.

**Comment by itzg** [comment]
FWIW this is the one that worked for me

https://github.com/itzg/mc-modpacks-starter/blob/master/rlcraft/docker-compose.yml

**Comment by brutalgg** [comment]
I'll try your exact compose @itzg but I'm pretty sure I had a similar one which was running into the error detailed in issue #743. Either way I've fixed the example compose to a working version for me with RLCraft actually being loaded this time.

**Comment by brutalgg** [comment]
> FWIW this is the one that worked for me
> 
> https://github.com/itzg/mc-modpacks-starter/blob/master/rlcraft/docker-compose.yml

I double checked this config using a unmodified zip download from curseforge and it resulted in the following loop.

```
[init] Unpacking FTB server modpack /modpacks/RLCraft_Server_Pack_1.12.2_Beta_v2.8.2.zip ...
..........
[init] Please make sure you are using the server version of the FTB modpack!
```

File: examples/docker-compose-rlcraft.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      VERSION: "1.12.2"
+      FORGEVERSION: "14.23.5.2855"
+      DIFFICULTY: "hard"
+      MAX_TICK_TIME: "-1"
+      VIEW_DISTANCE: "6"
+      ALLOW_FLIGHT: "true"
+      MEMORY: "4G"
+      GENERIC_PACK: "/modpacks/RLCraft_Server_Pack_1.12.2_Beta_v2.8.2.zip"
```
Comment (itzg) [inline examples/docker-compose-rlcraft.yml:20]: Ah yes, I remember this now being the solution for RLCraft since their server modpack is "broken".

**Review by itzg — APPROVED** [review]
Thanks!

**Pull request merged**